bottlenet.go

```diff
@@ -1,3 +1,21 @@
+// +build go1.13
+
+/*
+ * MinIO Cloud Storage, (C) 2016, 2017, 2018, 2019, 2020 MinIO, Inc.
```

Comment:
you need to add `Bottlenet` - also perhaps the license is AGPLv3? or is it Apache?

Reply:
Also license should state just this year since this is only committed this year. 

```
Bottlenet, (C) 2020 MinIO, Inc.
```

Reply:
Should be like this 

```
/*
 * Bottlenet (C) 2020 MinIO, Inc.
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Affero General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU Affero General Public License for more details.
 *
 * You should have received a copy of the GNU Affero General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
```